lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js

```diff
@@ -595,4 +598,78 @@
         }
         return o;
     });
+
+    platform.registerFunction(platform.FunctionKind.Math_min, function (value1, value2, ...values) {
```

Comment:
Looks like we could omit `...values` as we access them via `arguments` instead

Reply:
1a82165a5e48c87f6559f5c109d3810c62b0800e